aws/resource_aws_lambda_layer_version_test.go

```diff
@@ -345,7 +345,7 @@ resource "aws_lambda_layer_version" "lambda_layer_test" {
   filename   = "test-fixtures/lambdatest.zip"
   layer_name = "%s"
 
-  compatible_runtimes = ["nodejs8.10", "nodejs6.10"]
```

Comment:
The `TestAccAWSLambdaLayerVersion_compatibleRuntimes` test that references this test configuration is currently expecting two values for the `compatible_runtimes` argument:

https://github.com/terraform-providers/terraform-provider-aws/blob/feacc804717a491202318f1ab217f5c76fa049e1/aws/resource_aws_lambda_layer_version_test.go#L168

Can you please add back a second runtime, e.g. `nodejs10.x`

Reply:
Addressed: https://github.com/terraform-providers/terraform-provider-aws/pull/9341/commits/95128f1568196332ae8fd16297269bb6be4c6bc8